src/OpenCV/Photo.hs

```diff
@@ -244,3 +259,59 @@ fastNlMeansDenoisingColoredMulti h hColor templateWindowSize searchWindowSize sr
         | c'srcVecLength `mod` 2 == 1 = c'srcVecLength
         | otherwise                   = c'srcVecLength - 1
     c'imgToDenoiseIndex = (c'temporalWindowSize - 1) `div` 2
+
+{- | Preform denoise_TVL1
+
+Example:
+
+@
+
+eachChannel f img = unsafeCoerceMat . matMerge <$> V.mapM f (matSplit img)
```

Comment:
This gives a warning when compiling `opencv-doc-images`:

> Top-level binding with no type signature

And since this is the example code, a small explanation of what `eachChannel` does would be useful. Alternatively, if the function is generally useful we could move it somewhere to the core modules.